api/controller/utils/qrCodeUtil.js

```diff
@@ -0,0 +1,13 @@
+var qrcode = require('qrcode');
+
+exports.generateQRCode = async function (otpAuth) {
+
+	try {
+		var QRCodeImageUrl = await qrcode.toDataURL(otpAuth);
+		return {image: `<img src='${QRCodeImageUrl}' alt='qr-code-img' />`};
+	} catch (error) {
+		console.log('Could not generate QR code', error);
+		return;
+	}
+    
+};
```

Comment:
Move this to dashboard utils and improve response handling.

Reply:
I moved this to a seperate file as I had to use es6+ syntax and having it in dashboard controllers produced eslint errors hence I moved it to api/utils as I did the same with translated mail utility.

Reply:
You can use `Promises` to fix the linting issue instead of `async/await`.